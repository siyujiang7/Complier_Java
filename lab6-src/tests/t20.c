void func(long b, long c) {
  long a;
  for (a = 0; a < 100; a = a + 1) {
    b = b + c;
  }
  printf("%ld %ld %ld\n",
      a, b, c);
  return;
}
long main() {
  long b, c;
  b = 1;
  c = 7;
  func(b, c);
}
