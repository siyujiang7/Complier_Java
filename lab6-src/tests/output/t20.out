100 701 7
